cmake-variants.yaml

```diff
@@ -0,0 +1,27 @@
+buildType:
+  default: Develop
+  choices:
+    Develop:
+      short: Develop
+      long: Emit debug information but also optimize
+      buildType: Develop
+    Debug:
+      short: Debug
+      long: Emit debug information and don't optimize
+      buildType: Debug
+    Release:
+      short: Release
+      long: Optimize generated code
+      buildType: Release
+board:
+  # Fill name of your targer which has to corespond to
+  # target name in custom_targets.json5 file
+  default: L452RE_SIMPLE
+  choices:
+    YOUR_MBED_TARGET:
+      short: L452RE_SIMPLE
+      settings:
+        MBED_TARGET: L452RE_SIMPLE   
+        # Fill your upload method
+        # variants are: NONE, MBED, STLINK, JLINK, PYOCD, OPENOCD, STM32CUBE
```

Comment:
LinkServer: 🥺

Reply:
Good point, it was my old helper line and i forgot also Picotool and Arduino bossac.